xapian-letor/tests/api_letor.cc

```diff
@@ -220,8 +220,8 @@ DEFINE_TESTCASE(createfeaturevector, generated)
     TEST_EQUAL(mset.size(), 2);
     auto fv = fl.create_feature_vectors(mset, Xapian::Query("lions"), db);
     TEST_EQUAL(fv.size(), 2);
-    TEST_EQUAL(fv[0].get_fcount(), 19);
-    TEST_EQUAL(fv[1].get_fcount(), 19);
+    TEST_EQUAL(fv[0].get_fcount(), 20);
+    TEST_EQUAL(fv[1].get_fcount(), 20);
```

Comment:
These test changes should really be in the commit that made them necessary to maintain passing tests.